nailgun/entity_fields.py

```diff
@@ -265,4 +266,4 @@ class URLField(StringField):
 
     def gen_value(self):
         """Return a value suitable for a :class:`URLField`."""
-        return gen_url()
+        return gen_url(subdomain=gen_alpha())
```

Comment:
@omaciel There's two issues:

* `gen_url`, `gen_email` and possibly others generate real values that might be in use in the wild. For example, `gen_url` might spit out [test.com](https://www.test.com/).
* `gen_url` doesn't produce especially random values. That's why we're calling `gen_url(subdomain=gen_alpha())` instead of `gen_url()` here in NailGun.

omaciel/fauxfactory#26 is describes how `gen_email` produces "real" values. There aren't issues for the other methods that produce "real" values. There's not an issue for `gen_url`'s lack of randomness.